app/uk/gov/hmrc/helptosave/connectors/EligibilityCheckConnector.scala

```diff
@@ -45,16 +48,18 @@ class EligibilityCheckConnectorImpl @Inject() (http: WSHttp, metrics: Metrics, p
   def url(nino: String): String =
     s"$itmpBaseURL/help-to-save/eligibility-check/$nino"
 
-  override def isEligible(nino: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[EligibilityCheckResult] =
-    EitherT[Future, String, EligibilityCheckResult](
+  type EitherStringOr[A] = Either[String, A]
+
+  override def isEligible(nino: String)(implicit hc: HeaderCarrier, ec: ExecutionContext): Result[Option[EligibilityCheckResult]] =
+    EitherT[Future, String, Option[EligibilityCheckResult]](
       {
         val timerContext = metrics.itmpEligibilityCheckTimer.time()
 
         http.get(url(nino), desHeaders)(hc.copy(authorization = None), ec)
           .map { response ⇒
             val time = timerContext.stop()
 
-            response.status match {
+            val r: Option[Either[String, EligibilityCheckResult]] = response.status match {
```

Comment:
`r` isn't very descriptive - `result` maybe?

Reply:
changed to res, as already have a val result in scope